tendrl/ceph_integration/tests/fixtures/client.py

```diff
@@ -0,0 +1,11 @@
+class Client():
```

Comment:
This style of class gives a pep8 error. Extend from `object`. Applies to all the instances.